projects/frontend/data-pipelines/gui/angular.json

```diff
@@ -0,0 +1,220 @@
+          "options": {
+            "outputPath": "dist/ui",
+            "index": "projects/ui/src/index.html",
+            "main": "projects/ui/src/main.ts",
+            "polyfills": "projects/ui/src/polyfills.ts",
+            "tsConfig": "projects/ui/tsconfig.app.json",
+            "allowedCommonJsDependencies": [
+              "cron-parser",
+              "dagre",
+              "webcola"
+            ],
+            "assets": [
+              "projects/ui/src/favicon.ico",
+              "projects/ui/src/assets",
+              {
+                "glob": "clr-ui*.min.*",
+                "input": "node_modules/@clr/ui",
+                "output": "assets/css"
+              },
+              {
+                "glob": "**/*",
+                "input": "node_modules/@vdk/shared/assets",
+                "output": "/assets/"
+              },
+              {
+                "glob": "**/*",
+                "input": "node_modules/@vdk/data-pipelines/assets",
+                "output": "/assets/"
+              }
+            ],
+            "styles": [
+              "node_modules/@clr/icons/clr-icons.min.css",
+              "node_modules/@angular/cdk/overlay-prebuilt.css",
+              "projects/ui/src/styles.scss"
+            ],
+            "scripts": [
+              "node_modules/@clr/icons/clr-icons.min.js"
+            ]
+          },
+          "configurations": {
+            "development": {
+              "sourceMap": {
+                "scripts": true,
+                "vendor": true,
+                "styles": true
+              },
+              "vendorChunk": true,
+              "extractLicenses": false,
+              "buildOptimizer": false,
+              "optimization": false,
+              "namedChunks": true,
+              "preserveSymlinks": true
+            },
+            "production": {
+              "fileReplacements": [
+                {
+                  "replace": "projects/ui/src/environments/environment.ts",
+                  "with": "projects/ui/src/environments/environment.prod.ts"
+                }
+              ],
+              "sourceMap": false,
+              "vendorChunk": false,
+              "extractLicenses": true,
+              "buildOptimizer": true,
+              "optimization": true,
+              "namedChunks": false,
+              "outputHashing": "all",
+              "budgets": [
+                {
+                  "type": "initial",
+                  "maximumWarning": "5mb",
+                  "maximumError": "10mb"
+                },
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb",
+                  "maximumError": "10kb"
+                }
+              ]
+            }
+          },
+          "defaultConfiguration": "development"
+        },
+        "serve": {
+          "builder": "@angular-devkit/build-angular:dev-server",
+          "options": {
+            "host": "localhost.vmware.com",
```

Comment:
I'm not completely sure if it is fine to use -> 'vmware.com' in the URL for the opensource project. 

Please, double check if this can work for more generic "host" config(e.g. 'localhost.vdk.com', or similar). 

Note:  As far as I can recall, the CSP Authentication configs may need some tweaks to make this working.

Reply:
Is there a reason to be fancy? Why not just use `localhost:4200`?

Reply:
If I recall correctly the CSP authentication mechanism did not allow localhost. 

But this has to be double-checked. And yes, if localhost:4200 works fine, I'm also fine.

Reply:
There already is localhost.vmware.com [published](https://github.com/vmware/versatile-data-kit/blob/1a0dcff00fb69ac1cd78b1a7eeeec9ca1e20ea22/projects/control-service/projects/base/src/test/resources/application.properties#L15),
the proposal is to create a ticket and revise all occurrences if needed,
in a separate to this PR effort. 

Please share any thoughts you have on this topic?

Reply:
Recalling my local tests, there is auth policies on access control 
- in regards to deleted vmware.com top domain.

Reply:
yes, please create se separate ticket to have this resolved.